src/pages/components/externalPages/ExternalPageTemplate.js

```diff
@@ -5,7 +5,7 @@ import FooterContent from '../footerSection/FooterContent';
 const ExternalPageTemplate = ({ title, children }) => {
   return (
     <Fragment>
-      <header className='bg-main-backgound h-[100px] border-b border-brown-900 flex items-center sticky top-0'>
```

Comment:
Please don't remove the bg color, just change to bg-orange-100

Reply:
I accidentally remove it sorry 😅